lib/litani_report.py

```diff
@@ -570,11 +562,27 @@ def render(run, report_dir, pipeline_depgraph_renderer):
                     MemoryTraceRenderer.render(
                         temporary_report_dir / pipe["url"], env, job, gnuplot)
 
+                tags = job["wrapper_arguments"]["tags"]
+                description = job["wrapper_arguments"]["description"]
+                if tags and "front-page-text" in tags:
+                    if "stdout" in job:
```

Comment:
I think `if "stdout" in job and job["stdout"]`, since even if the key exists, its value may be None as per the schema.